src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Json/XmlJsonWriter.cs

```diff
@@ -23,14 +23,45 @@ internal class XmlJsonWriter : XmlDictionaryWriter, IXmlJsonWriterInitializer
         private const char WHITESPACE = ' ';
         private const char CARRIAGE_RETURN = '\r';
         private const char NEWLINE = '\n';
-        private const char BACKSPACE = '\b';
-        private const char FORM_FEED = '\f';
-        private const char HORIZONTAL_TABULATION = '\t';
         private const string xmlNamespace = "http://www.w3.org/XML/1998/namespace";
         private const string xmlnsNamespace = "http://www.w3.org/2000/xmlns/";
 
         // This array was part of a perf improvement for escaping characters < WHITESPACE.
-        private static readonly string[] s_escapedJsonStringTable = CreateEscapedJsonStringTable();
+        private static readonly string[] s_escapedJsonStringTable =
+        {
+            "\\u0000",
+            "\\u0001",
+            "\\u0002",
+            "\\u0003",
+            "\\u0004",
+            "\\u0005",
+            "\\u0006",
+            "\\u0007",
+            "\\b",
+            "\\t",
+            "\\n",
+            "\\u000b",
+            "\\f",
+            "\\r",
+            "\\u000e",
+            "\\u000f",
+            "\\u0010",
+            "\\u0011",
+            "\\u0012",
+            "\\u0013",
+            "\\u0014",
+            "\\u0015",
+            "\\u0016",
+            "\\u0017",
+            "\\u0018",
+            "\\u0019",
+            "\\u001a",
+            "\\u001b",
+            "\\u001c",
+            "\\u001d",
+            "\\u001e",
+            "\\u001f"
+        };
```

Comment:
Do we have sufficient tests to verify that the values are correct and stay correct? Previously, the table was generated and it gave us reasonable confidence the values are what they should be.

Reply:
Good point. I added more tests to ensure the entire table is covered.